src/DependencyInjection/DI.External.Tests/test/SkippableDependencyInjectionSpecificationTests.cs

```diff
@@ -11,7 +11,7 @@ public abstract class SkippableDependencyInjectionSpecificationTests: Dependency
 
         protected sealed override IServiceProvider CreateServiceProvider(IServiceCollection serviceCollection)
         {
-            foreach (var stackFrame in new StackTrace(1).GetFrames().Take(2))
+            foreach (var stackFrame in new System.Diagnostics.StackTrace(1).GetFrames().Take(2))
```

Comment:
Why is this change necessary?

Reply:
This namespace shouldn't be needed but I had the strange behavior that I couldn't get to resolve StackTrace unless I fully specified it. Using statement kept being ignored.  Both intellisense and the compiler complained about it.  I had the option enable to let VS use preview tooling and the latest preview4 build was just installed.

Reply:
This shouldn't be required.

Reply:
Removed it from the code.  Intellisence still complains but the code builds and the tests run ok.

Reply:
It's possible Intellisense was finding an internal StackTrace member in some other namespace...  Still intellisense and compile ought to be using the same algorithm.  If you can reproduce it we can file a bug against roslyn.